google/resource_compute_region_instance_group_manager_test.go

```diff
@@ -204,6 +204,32 @@ func TestAccRegionInstanceGroupManager_autoHealingPolicies(t *testing.T) {
 	})
 }
 
+func TestAccRegionInstanceGroupManager_distributionPolicy(t *testing.T) {
+	t.Parallel()
+
+	var manager computeBeta.InstanceGroupManager
+
+	template := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	igm := fmt.Sprintf("igm-test-%s", acctest.RandString(10))
+	zones := []string{"zones/europe-west3-a", "zones/europe-west3-b"}
```

Comment:
is it how people use zones in tests for this provider? Hardcoded vs Env vars?

Reply:
That looks right. He just  defines it for the IGM  resource to use for the distribution_policy argument.  All  resource arguments are normally hardcoded in tests